index.html

```diff
@@ -227,10 +227,8 @@
 };
 
 interface IdleDeadline {
-  readonly attribute DOMHighResTimeStamp deadline;
+  double timeRemaining();
```

Comment:
I thought of that, but according to the DOMHighResTimeStamp spec it is "a time value measured relative to the navigationStart", which is not the case for this return value (this return value is a time delta, not an absolute time).